lib/ansible/plugins/connection/ssh.py

```diff
@@ -167,9 +167,9 @@
               version_added: '2.7'
           cli:
             - name: ssh_extra_args
-      retries:
+      reconnection_retries:
           description: Number of attempts to connect.
-          default: 3
+          default: 0
```

Comment:
Why not keep the default 3?

Reply:
The actual default from `base.yml` was 0 before.

https://github.com/ansible/ansible/commit/935528e22e5283ee3f63a8772830d3d01f55ed8c#diff-3f3fbfd37f8f95b3e5f26617feb866a00e1cddb44fad1522321927d42349b877L168

It's been 3 [in the connection plugin docs for a while](https://github.com/ansible/ansible/commit/f921369445900dcc756821e40c9257d5359087af#diff-38cec806ea1a1ee7c3a286c7865334ecfba7ccc49e21d4e8fb8ec1b17938fda6R90) but didn't actually do anything. So this is just putting it back the way it was.

I wonder if keeping this at 0 would prevent account lockout if a password is incorrect.